spec/modules-spec.js

```diff
@@ -47,6 +47,20 @@ describe('third-party module', function () {
       })
     })
   })
+
+  describe('global variables', function () {
+    describe('process', function () {
+      it('can be declared in a module', function () {
+        require(path.join(fixtures, 'module', 'declare-process.js'))
```

Comment:
Can you wrap the `require` in a `assert.doesNotThrow` so if it does fail ever the message will be reported better?